src/bindings/python/fluxacct/accounting/jobs_table_subcommands.py

```diff
@@ -88,7 +88,9 @@ def write_records_to_file(job_records, output_file):
             )
 
 
-def fetch_job_records(job_records):
+# Convert the results of a query to the jobs table to a readable string
+# that can either be output to stdout or written to a file.
```

Comment:
This is the style police 👮‍♂️ 🚔 you have violated [the law](https://peps.python.org/pep-0257/)

Reply:
👐 i swear i'm innocent!

I will change the style of the function descriptions that I've created in this PR. If it's okay, I'll open a separate one to fix the descriptions in the rest of the repo.